sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -1857,6 +1857,18 @@ object GpuOverrides extends Logging {
           // passing the already converted strf string for a little optimization
           GpuFromUnixTime(lhs, rhs, strfFormat)
       }),
+    expr[FromUTCTimestamp](
+      "Interpret the given timestamp as in UTC and render that time in the given time zone",
+      ExprChecks.binaryProject(TypeSig.STRING, TypeSig.STRING,
+        ("timestamp", TypeSig.TIMESTAMP, TypeSig.TIMESTAMP),
+        ("timezone", TypeSig.lit(TypeEnum.STRING)
+          .withPsNote(TypeEnum.STRING, "A limited number of time zones are supported"),
+          TypeSig.STRING)),
+      (a, conf, p, r) => new BinaryExprMeta[FromUTCTimestamp](a, conf, p, r) {
```

Comment:
There needs to be a tagForGpu override that checks if the timezone is equivalent to UTC.